buildSrc/src/main/kotlin/Dependencies.kt

```diff
@@ -8,6 +8,7 @@ object Versions {
     const val versionCode = 1
     const val versionName = "1"
     const val gradleBuildTool = "4.2.1"
+    const val dataBinding = "3.2.0-alpha11"
```

Comment:
can you also remove this line, as its not related to the changes needed :)

Reply:
> can you also remove this line, as its not related to the changes needed :)

Done